WINDOWS_SETUP.md

```diff
@@ -76,50 +77,50 @@ export PATH=$PATH:$DART
 
 Exit out of nano with `CTRL-x & CTRL-y`.
 
-11. At this point we'd be required to restart our Ubuntu machine. But since we're use WSL 2 we can just restart our distro instance. Close the app, and re-open it.
+11. At this point we'd be required to restart our Ubuntu machine. But since we're use WSL 2 we can just restart our distro instance. Close the app, and then re-open it.
```

Comment:
"But since we're **use**..." should be changed to "But since we're **using**..."